ansible_collections/arista/avd/roles/eos_validate_state/tasks/main.yml

```diff
@@ -20,6 +20,8 @@
     - "{{ eos_validate_state_dir }}"
   delegate_to: localhost
   run_once: true
+  tags:
+    - always
```

Comment:
Maybe align with the reports task, so always and reports. So if run with skip-tags reports this is not run

Reply:
nevermind.